### PR TITLE
Add sample_rate configuration

### PR DESCRIPTION
**Pull request by dcramer**

### CODE REVIEW

**Comment by dcramer** [comment]
This should mimic the behavior in Java, which we should also add to client-dev spec.

**Comment by dcramer** [comment]
Gonna see if i can add some simple tests around this, though not sure how reasonable it is to mock this.

File: lib/Raven/Client.php [hunk 1]
```diff
@@ -971,6 +973,11 @@ public function send(&$data)
             return;
         }
 
+        // should this event be sampled?
+        if (rand(1, 100) / 100.0 > $this->sample_rate) {
```
Comment (nokitakaze) [inline lib/Raven/Client.php]: Rand?! REALLY?!

Comment (dcramer) [inline lib/Raven/Client.php]: What's the concern?

Comment (nokitakaze) [inline lib/Raven/Client.php]: Nothing. I just will replace it by mt_rand in #414

**Pull request merged**